mysql-test/suite/binlog_encryption/t/binlog_rotate_key_check_privileges.test

```diff
@@ -0,0 +1,15 @@
+--source include/not_embedded.inc
+# This test is to check that only SUPER users can rotate_system_key
+
+CREATE USER 'regular_user'@'localhost' IDENTIFIED BY 'pass';
+
+--connect (regular_user_con,localhost,regular_user,pass,test)
+
+--error ER_SPECIFIC_ACCESS_DENIED_ERROR
+SELECT rotate_system_key("percona_binlog");
+
+--connection default
+SELECT rotate_system_key("percona_binlog");
+
+--disconnect regular_user_con
```

Comment:
--source include/wat_until_count_sessions.inc anywhere between disconnect and the end of testcase

Reply:
Done